proposals/nnnn-main-menu-updating.md

```diff
@@ -162,14 +162,14 @@ The HMI will have to support this feature and tell Core when to request menu upd
 ### Manager-Level Updates
 The app library Menu Manager will need updates to work properly with this system. The menu manager will have to know when dynamic menu features are available and send only the appropriate data. 
 
-When `WindowCapability.supportsDynamicMenus` is `true` the menu managers will not send artworks until `UpdateCommand` or `UpdateSubMenu` is called with `UpdateArtwork`. At that point, it will have to upload the artwork and once that's finished, send a new `AddCommand` or `AddSubMenu` with the file name in the `image` parameter. 
+When `WindowCapability.supportsDynamicMenus` is `true` the menu managers will not send artworks until `UpdateCommand` or `UpdateSubMenu` is called with `UpdateArtwork`. At this point, the menu manager with `WindowCapability.supportsDynamicMenus = true` will have to upload the artwork and once that's finished, send a new `AddCommand` or `AddSubMenu` with the file name in the `image` parameter. 
 
-It will also not send sub-menus until `UpdateSubMenu` is called with `UpdateSubCells` as `true`. At that point, it will have to send `AddCommand` for all the submenu cells (while still respecting).
+The menu manager with `WindowCapability.supportsDynamicMenus = true` will also not send sub-menus until `UpdateSubMenu` is called with `UpdateSubCells` as `true`. At that point, it will have to send `AddCommand` for all the submenu cells (while still respecting).
 
 However, I do not believe any new APIs will need to be added to the manager system, and therefore this does not need to be designed in this proposal.
 
 ## Potential downsides
-This adds complexity to the HMI, which will have to keep track of which menu items exist, have artwork available but not on the system, and so forth. It will need to determine when it wants to send the `UpdateXXX` notifications. e.g. Should it send when the cell / sub-menu is on screen, or at some point before? This is a moderate complexity increase. The internal database of which cells need artwork is not too difficult due to the `AddCommand` / `AddSubMenu` not setting an image name until the artwork is actually uploaded.
+This adds complexity to the HMI. The HMI will have to keep track of which menu items exist, have artwork available but not on the system, and so forth. The HMI will need to determine when it wants to send the `UpdateXXX` notifications. For example, should it send when the cell / sub-menu is on screen, or at some point before (this is a moderate complexity increase)? The internal database of which cells need artwork is not too difficult due to the `AddCommand` / `AddSubMenu` not setting an image name until the artwork is actually uploaded.
```

Comment:
Please update 
for example, should it send when the cell / sub-menu is on screen, or at some point before (this is a moderate complexity increase)?

to

For example, should it send when the cell / sub-menu is on screen, or at some point before? (this would be a moderate complexity increase).

Reply:
Your suggestion would make a full sentence out of the parenthetical, but I already changed it to parenthesis based on a previous review, so instead I changed the sentence structure a bit. I hope that's okay.